tox.ini

```diff
@@ -1,10 +1,17 @@
 [tox]
+skipsdist = True
 envlist = py{27,35}, py{27,35}-marionette, flake8
 
 [testenv]
-passenv = DISPLAY PYTEST_ADDOPTS
+usedevelop = True
+passenv = DISPLAY PYTEST_ADDOPTS PYTEST_BASE_URL SAUCELABS_API_KEY SAUCELABS_USERNAME
 deps = -rrequirements/tests.txt
-commands = pytest --ignore tests/marionette {posargs}
+commands = python setup.py develop \
```

Comment:
We shouldn't need to run python setup.py develop. Also, the `\` concatenates, so this command is bleeding into the next one. I think we only need one command here, but it can be across multiple lines for readability.